saleor/product/migrations/0139_description_vector_search.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 3.1.5 on 2021-01-14 08:31
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.contrib.postgres.search import SearchVector
+from django.db import migrations, models
+
+
+def parse_description_json_to_string(description):
+    string = ""
+    blocks = description.get("blocks")
+    if not blocks or not isinstance(blocks, list):
+        return ""
+    for block in blocks:
+        block_type = block["type"]
+        if block_type == "list":
+            for item in block["data"].get("items"):
+                if not item:
+                    continue
+                string += item
+        else:
+            text = block["data"].get("text")
+            if not text:
+                continue
+            string += text
+    return string
+
+
+def migrate_description_into_description_search(apps, schema):
+    Product = apps.get_model("product", "Product")
+    for product in Product.objects.iterator():
+        product.description_search = parse_description_json_to_string(
```

Comment:
@maarcingebala @d-wysocki  Are you sure that the name `description_search` is the best one? Personally I would say that this is a raw description which we use to search(but we still can use to differen things).

Reply:
Also, what is the difference between content from `Product.description_search` and `Product.description`?

Reply:
`description` is formatted content for editor js (JSON) and `description search` is parsed JSON content to a string to allow using SearchVector.

This field for now is internal field only for text search purpose.

Reply:
That's true the field could be used for different things. Maybe `description_plaintext`?

Reply:
`description_search` would suggest that contains something related only for searching and it contains only the raw description text. `description_plaintext` or `description_raw` or `description_str`, all of them would be better imo.

Reply:
ok, I will change it